lib/streamlit/metrics_util.py

```diff
@@ -0,0 +1,76 @@
+import os
+import platform
+import subprocess
+import threading
+import uuid
+
+from typing import Optional
+
+from streamlit import file_util
+
+def _get_machine_id_v1():
+    """Get the old machine id"""
+    if (
+        platform.system() == "Linux"
+        and os.path.isfile("/etc/machine-id") == False
+        and os.path.isfile("/var/lib/dbus/machine-id") == False
+    ):
+        print("Generate machine-id")
```

Comment:
Is this print statement supposed to be here?

Reply:
I had copied this code unchanged from it's previous location. I'll remove the print statement.